### DISCUSSION TITLE
Filter Action based on Image/Object

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#6917] [Q&A] [answered]
I was wondering if there was a way to provide a "Filter Action" based on an image instead of a word, etc.  There is a photo feed which at times contains stories or text content which I would like to filter out, if able.  

At first, I was not sure if the image was a ASCii character which I could include ot some regex, but looking at the development tools in Chrome, it appear as an image (of a floppy disk) in the DIV seciton of the article.  Just wondering if I may be missing a way to mark these articles as read using the Filter Action section.  The sample below is one of many users who post this type of content, so I woudl be unable to filter on the "@UserName" section.  

Below is what I see in the Freshrss UI and a clip from the Developer Tools:

![Screenshot 2024-10-18 at 7 54 35 AM](https://github.com/user-attachments/assets/e6b2b5d7-3113-4900-855f-b62c7beb64c7)

![Screenshot 2024-10-18 at 7 56 40 AM](https://github.com/user-attachments/assets/f3cb5df4-f2a1-47d7-8001-8e40c644d197)

**Comment by Alkarex** [comment]
Hello,
The floppy-disk icon 💾 is one that we add to the UI in the case when the article contains an attachment (such as an audio file or something else) to be able to access it.

But I am not sure to understand your question. Maybe you could provide the link of the feed and an example.

Yes, you can automatically mark as read an article based on the presence or absence of an `<img ` tag with a given URL pattern. Not the 💾 though, which is added by FreshRSS and not a part of the original article.

**Reply by quella** [reply]
@Alkarex Thank you as always for the detailed resolution.  Yes, I would rather just filter on anything that does not have an image (any other content in the feed should be "Marked as Read" so I only receive images in the feed.  What would need to be added to the Filter to perform such a task?  I have always only put text in the "Filter Actions" section.  Is there a different place to populate an article not containing an "<IMG" tag?

**Reply by Alkarex** [reply, marked as answer]
You can search for articles that do not have any image in their body with `!/<img/i` (since FreshRSS 1.25)
However, there is no way at the moment to search for enclosures (attachments). That would be a nice extra feature

**Reply by quella** [reply]
Sir, thank you for your assistance.  I will have a look at the documentation on some of the more advanced filter options and play around.  Thanks again for a great product which gets better and better with each release.

**Reply by Alkarex** [reply]
https://freshrss.github.io/FreshRSS/en/users/10_filter.html#regex (since FreshRSS 1.25)
